app/src/main/java/kr/co/connect/boostcamp/livewhere/data/SharedPreferenceStorage.kt

```diff
@@ -0,0 +1,38 @@
+package kr.co.connect.boostcamp.livewhere.data
+
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.core.content.edit
+import kotlin.properties.ReadWriteProperty
+import kotlin.reflect.KProperty
+
+interface SharedPreference {
+    var uuid: String?
+}
+
+class SharedPreferenceStorage(context: Context) : SharedPreference {
+    private val prefs = context.applicationContext.getSharedPreferences(PREF, Context.MODE_PRIVATE)
+
+    override var uuid: String? by UuidPreference(prefs, PREF_UUID, null)
```

Comment:
uuid값은 한번 저장되면 바뀌지 않는 값인데, val로 처리하는 것이 어떨까 싶어요. 
```suggestion
    override val uuid: String? by UuidPreference(prefs, PREF_UUID, null)
```